### DISCUSSION TITLE
How should I configure VS Code for debugging?

### DISCUSSION THREAD

**Discussion by TomMarks** [elgatosf/elgato#134] [Help] [answered]
I'd like to debug my stream deck plugin, but I'm not able to get the plugin to work right.  How do I configure VS Code to debug the plugin on the stream deck?  Do I need to attach to a process?

**Comment by GeekyEggo** [comment, marked as answer]
Hi Tom, that's correct, you'll need to attach to the process — you can find more about debugging in VS Code in our [new documentation](https://docs.elgato.com/streamdeck/sdk/introduction/your-first-changes#debugging). We are also planning to make this experience better in VS Code in the future.

**Comment by TomMarks** [comment]
Thanks for the info - I'll take a look.

**Comment by TomMarks** [comment]
Thanks again, GE.  I'm debugging!
